library/datasort.h

```diff
@@ -57,6 +57,11 @@ struct datasort_chunk {
 	uint64_t			index_size;
 	/* Set to 1 if chunk came from sorted bctl */
 	uint8_t				already_sorted;
+	/* Set to 1 if chunk is just a view from base ctl
+	 * View can be created only from sorted base ctl. Instead it doesn't make a sense
+	 * We need this field in destroying to distinguish cases when we own and don't own a file descriptor fd
+	 */
+	uint8_t 		 	base_view;
```

Comment:
(not mandatory) Now we can use `flags` field instead of `already_sorted` and `base_view`.